tempesta_fw/t/functional/README.md

```diff
@@ -0,0 +1,197 @@
+# Functional Tests for TempestaFW
+
+## Recommended configuration
+
+Running tests during development process can cause crashes to TempestaFW.
+Since TempestaFW is implemented as a set of kernel modules it is not convenient
+to run testing framework on the same host. It is recommended to run testing
+framework on a separated host.
+
+Recommended test-beds:
+
+- Fully distributed. 4 different hosts with their own roles are used. This
```

Comment:
I think in most cases the test suite will be run on the same VM, just to verify that Tempesta FW passes all the tests. The distributed test is certainly useful, but for development tasks only. Thus, it's better to use single VM installation as a default configuration.